evalml/tests/automl_tests/parallel_tests/test_cf_engine.py

```diff
@@ -213,7 +213,18 @@ def test_submit_evaluate_job_single(
         )
 
 
-@pytest.mark.parametrize("pool_type", ["threads", "processes"])
+@pytest.mark.parametrize(
+    "pool_type",
+    [
+        pytest.param(
+            "threads",
+            marks=pytest.mark.xfail(
+                reason="Incompatibility between Woodwork 0.17.x and Concurrent Futures.",
```

Comment:
Seems confusing here, since we should be using WW >= 0.18, not 0.17?

Reply:
Reverted!